contracts/test/DepositStub.sol

```diff
@@ -6,6 +6,13 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 import "../RiskManagerV1.sol";
 
+/// @dev tBTC v1 deposit interace with a subset of functions used by unit and
+///      system tests. For more information about tBTC Deposit, please see:
+///      https://github.com/keep-network/tbtc/blob/solidity/v1.1.0/solidity/contracts/deposit/Deposit.sol
+interface ITBTCDeposit is IDeposit {
```

Comment:
Shouldn't `DepositStub` contract declared below inherit from `ITBTCDeposit`?
It has `notifyRedemptionSignatureTimedOut` function.

Reply:
https://github.com/keep-network/coverage-pools/pull/124/commits/fb8e4ad1cc1efae2b923e7e53bf8031bf0ab34ae